eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           NuGetCacheKey: Test
       - script: >-
           dotnet test eng/service.proj --filter "(TestCategory!=Manually) & (TestCategory!=Live)" --framework $(TestTargetFramework)
-          --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal"
+          --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal" --blame-crash
```

Comment:
Should we also specify `--blame-crash-dump-type` to get a full dump?

Reply:
Not sure. The docs indicate that blame-crash will already collect the dump.

Reply:
My guess is that it's a mini dump by default. We should get a full if we can.

Reply:
I guess we will see!

Reply:
![](https://media2.giphy.com/media/qFNnOKIs6tBD2/giphy.gif)

Reply:
Wouldn't we want to use a minidump if it proves sufficient?

Reply:
I'd just rather get all the info the first time, just in case :) The test logs already give us a basic stack, but I'm guessing we'll want to poke around a bit more.